lib/network/modules/components/nodes/Cluster.js

```diff
@@ -4,17 +4,16 @@ import Node from '../Node'
  * A Cluster is a special Node that allows a group of Nodes positioned closely together
  * to be represented by a single Cluster Node.
  *
- * @class Cluster
+ * @param {Object} options
```

Comment:
This block is parameters for the constructor; hence, I expect them to be defined there.